docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+This SDK supports a minimum Android API of 23 or higher.
+Android apps can be written in Kotlin or Java 8 or higher.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
```

Comment:
Custom is the only option right now, but added a section here in case that changes and we provide a pre-made button integration or something. Can eliminate this section if we don't need it.